backends/concrete-cpu/implementation/src/c_api.rs

```diff
@@ -1,6 +1,7 @@
 pub mod bootstrap;
 #[cfg(feature = "csprng")]
 pub mod csprng;
+pub mod encrypt;
```

Comment:
I'd rather call it something like `noise` or `noise_generation`
I guess we should also have `#[cfg(feature = "csprng")]`

Reply:
I thought we are calling modules in the CAPI depending on which module they implement, no?

Reply:
Indeed, I didn't notice
Makes sense